src/test/java/com/contact/application/controller/DeleteControllerTest.java

```diff
@@ -0,0 +1,17 @@
+package com.contact.application.controller;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DeleteControllerTest {
+
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    @Test
+    public void deleteContact() {
```

Comment:
Forget the test?

Reply:
No, I was running outta time, will add the test now.